python/practices/practice-3/src/path2/word_not_in_list.py

```diff
@@ -0,0 +1,71 @@
+"""
+Prints all the words in the book that are not in the word list.
+Reads the content of the ebook file and the word list file, and then finds and
+prints all the words from the ebook that do not appear in the word list.
+"""
+
+
+import os
+from constants.constants import EBOOK, WORD
+from helpers.file_reader import read_file
+
+
+def extract_words(content):
+    """
+    Extracts words from a content string and returns a list of words.
+
+    Args:
+        content (str): The content string containing words.
+
+    Returns:
+        list: A list of words extracted from the content.
+    """
+
+    word_list = []
+
+    for word in content.split():
+        cleaned_word = word.strip()
+
+        if cleaned_word:
+            word_list.append(cleaned_word)
```

Comment:
Can you refactor this code?
Ex: work_list = [word for work in content.split() if word.strip()]

Reply:
Done